WebDriverAgentLib/Categories/XCUIElement+FBIsVisible.m

```diff
@@ -42,7 +42,12 @@ - (BOOL)fb_isVisible
   if ([FBConfiguration shouldUseTestManagerForVisibilityDetection]) {
     return [(NSNumber *)[self fb_attributeValue:FB_XCAXAIsVisibleAttribute] boolValue];
   }
-  CGRect appFrame = self.application.frame;
+  CGRect appFrame;
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"11.0")) {
```

Comment:
I don't think this is iOS version dependant. Most likely Xcode version it was built with. Can you confirm?